### PR TITLE
[DOCS] Update Logs app image in Logs Monitoring doc

### PR DESCRIPTION
**Pull request by EamonnTP**
This PR updates the Logs app image in the "Logs Monitoring" guide and includes minor edits to the content.  

Preview: http://stack-docs_1074.docs-preview.app.elstc.co/guide/en/logs/guide/master/logs-ui-overview.html

### CODE REVIEW

File: docs/en/logs/logs-overview.asciidoc [hunk 1]
```diff
@@ -2,9 +2,9 @@
 [role="xpack"]
 == Logs monitoring overview
 
-Logs monitoring enables you to view logs from your infrastructure and identify problems in near real time.
+Logs monitoring enables you to view logs from your infrastructure to help identify problems in real-time.
 You can view logs from servers, containers, services, and so on.
-Then you can drill down to view more detailed information about an individual log entry, or you can seamlessly switch to view corresponding metrics, uptime information or APM traces where available. You can also use machine learning to automatically detect some kinds of log anomalies.
+Additionally, you can drill down to view detailed information about an individual log entry, or you can seamlessly switch to view corresponding metrics, uptime information, or APM traces where available. You can also use machine learning to detect specific log anomalies automatically.
 
 // Add links to metrics, uptime and APM when I have good places to link to.
```
Comment (bmorelli25) [inline docs/en/logs/logs-overview.asciidoc:10]: Can you also update the `image::images/logs-monitoring-architecture.png[]` to this image:

https://user-images.githubusercontent.com/20965076/67385217-fb338a00-f592-11e9-9bad-95d083dfed1a.png

Comment (bmorelli25) [reply docs/en/logs/logs-overview.asciidoc:10]: FYI, the updated image is from https://github.com/elastic/observability-dev/issues/378#issuecomment-545384684.

File: docs/en/logs/logs-ui-intro.asciidoc [hunk 2]
```diff
@@ -2,16 +2,16 @@
 [role="xpack"]
 == {logs-app}
 
-After you have set up logs ingestion and log data is streaming to {es}, you can view real-time and historic logs in a compact, customizable display.
+After setting up logs ingestion and log data is streaming to {es}, you can view real-time and historical logs in a compact, customizable display.
 The log data is correlated with metrics data in the {metrics-guide}/infrastructure-ui-overview.html[{metrics-app}], making it easier for you to diagnose problems.
 You can also view related application traces or uptime information where available.
 
-You can stream the logs in real time, or view historical logs from a specified time range.
+You can stream the logs in real-time, or view historical logs from a specified time range.
```
Comment (dedemorton) [inline docs/en/logs/logs-ui-intro.asciidoc:9]: I would not hyphenate.

The rule I've always heard is that real-time is the adjective form of real time (the noun). :-)

File: docs/en/logs/logs-installation.asciidoc [hunk 3]
```diff
@@ -42,7 +42,7 @@ You can install and configure {beats} shippers for most kinds of data directly f
 To install {beats} from {kib}, on the machine where you want to collect the data, open a {kib} browser window.
 In the *Add Data to Kibana* section, click *Add log data*.
```
[the remark this answers is not in the document]
Comment (EamonnTP) [reply docs/en/logs/logs-installation.asciidoc]: Thanks. Screenshot updated.

File: docs/en/logs/logs-overview.asciidoc (hunk 1 above)
Comment (EamonnTP) [reply docs/en/logs/logs-overview.asciidoc:10]: Screenshot updated.

File: docs/en/logs/logs-ui-intro.asciidoc (hunk 2 above)
Comment (EamonnTP) [reply docs/en/logs/logs-ui-intro.asciidoc]: Thanks @dedemorton! Good to know.

**Pull request merged**